config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
 
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :test
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = false
```

Comment:
Firing up delayed job worked for me as well.  Awesome.  I updated `config/environments/development.rb`.  This PR should be ready to go.